iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/MessagingClientTests.java

```diff
@@ -336,11 +332,11 @@ public void messagingClientTokenRenewalWithAzureSasCredential() throws Exception
         Tools.disposeTestIdentity(testDeviceIdentity, iotHubConnectionString);
     }
 
-    @Ignore // The IoT Hub instance we use for this test is currently offline, so this test cannot be run
     @Test
     @ContinuousIntegrationTest
     public void messagingClientValidatesRemoteCertificateWhenSendingTelemetry() throws InterruptedException
     {
+        Assume.assumeTrue(System.getProperty("os.name").toLowerCase().contains("linux"));
```

Comment:
Consider moving this to Tools.java where we have a method called "isAndroid" that is similar to what you have here.

```
public static boolean isLinux
{
    return System.getProperty("os.name").toLowerCase().contains("linux");
}
```

So that the test reads a bit more easily

```java
Assume.assumeTrue(Tools.isLinux());
```

Reply:
It also cuts down on the duplicated code you currently have